src/subscription/cart/content/SubscriptionContent.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from '@edx/frontend-platform/i18n';
+import { Badge } from '@edx/paragon';
+
+// TODO: Fix the a11y and dynamic headings
+const CartContents = ({ children }) => (
+  <div className="basket-section">
+    <h5 aria-level="2">
+      <FormattedMessage
+        id="payment.subscription.purchase.cart.heading"
+        defaultMessage="In your cart"
+        description="Describe what's in your cart"
+      />
+    </h5>
+    <div className="">
+      <div className="heading-badge-wrapper">
+        <h3 aria-level="2" className="mb-0">
+          <FormattedMessage
+            id="payment.subscription.purchase.cart.programTitle"
+            defaultMessage="Blockchain Fundamentals"
```

Comment:
Touching on this point again, if this PR will not include functionality, can you add a TODO comment for hard coded temp default message here? But if you plan to add it in this PR later before merge, disregard all of my comments about this :)

Reply:
There is already a TODO added in the component documentation